dcos/dcos-ui
# [Spike] Job form follow-up tasks

This PR addresses non-blocking feedback that came up in reviews of #3599 

- Adds a confirmation prompt before closing form modal [DCOS-49264](https://jira.mesosphere.com/browse/DCOS-49264)
- Adds translation for validation error messages [DCOS-49262](https://jira.mesosphere.com/browse/DCOS-49262)
- Fixes CPUs minimum value validation [DCOS-49268](https://jira.mesosphere.com/browse/DCOS-49268)
- Ensures validation messages end with periods [DCOS-49263](https://jira.mesosphere.com/browse/DCOS-49263)

## Testing

- Ensure you are prompted to confirm close when you click "Cancel"
- Try adding a fake translation here https://github.com/dcos/dcos-ui/compare/feature/DCOS-48308-new-jobs-form...nat/job-form-followup?expand=1#diff-758096c192079a23867a134b9aa82082R141, change language to Chinese, and then submit the form with an empty CPU field and ensure you see the fake translation
- Try submitting form with value 0 for CPU and ensure message says minimum is 0.01 instead of 0.1

## Trade-offs

N/A

## Dependencies

N/A

## Screenshots

N/A
